be1-go/hub/standard_hub/message_handling.go

```diff
@@ -79,40 +79,38 @@ func (h *Hub) handleRootChannelPublishMesssage(sock socket.Socket, publish metho
 	return nil
 }
 
-// handleRootChannelPublishMesssage handles an incominxg publish message on the root channel.
+// handleRootChannelPublishMesssage handles an incoming publish message on the root channel.
 func (h *Hub) handleRootChannelBroadcastMesssage(sock socket.Socket,
 	broadcast method.Broadcast) error {
 
-	id := -1
-
 	jsonData, err := base64.URLEncoding.DecodeString(broadcast.Params.Message.Data)
 	if err != nil {
 		err := xerrors.Errorf("failed to decode message data: %v", err)
-		sock.SendError(&id, err)
+		sock.SendError(nil, err)
```

Comment:
Why put a nil in this function ?

Reply:
because broadcasts don't have a messageID. According to the comment of SendError() we can put nil when no messageID is available.
Also &id was a pointer to the literal 1, which is what SendError sends when given nil as messageID, I felt this implementation was more compliant with how the function was written.

Reply:
Thanks for the clarification